Gemfile

```diff
@@ -1,11 +1,12 @@
+# frozen_string_literal: true
```

Comment:
Let's delete this line and we are good to go!

Reply:
Done